fhir-path/src/main/java/com/ibm/fhir/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -470,6 +470,20 @@ private void pushContext(Collection<FHIRPathNode> context) {
             }
         }
 
+        private Collection<FHIRPathNode> repeat(List<ExpressionContext> arguments) {
```

Comment:
none of the other implementations here had comments and so I just sort of followed suit...

agree it would be nice to at least point to the definition of these operations in some javadoc though...will try adding it for a handful of them

Reply:
done